### PR TITLE
AM2320

### PR DESCRIPTION
**Pull request by danielbernalb**
Add AM2320 sensor
change %d to %u for hum and tmp in influxDbParseFields
Change Avarage for Average, cannot modify: gui.displaySensorAvarage(apm25)

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -123,30 +138,39 @@ void wrongDataState(){
...
 
-void avarageLoop(){
+void averageLoop(){
   if (v25.size() >= stime){
-    apm25 = getPM25Avarage();  // global var for display
-    apm10 = getPM10Avarage();
+    apm25 = getPM25Average();  // global var for display
+    apm10 = getPM10Average();
+
+////////////
```
Comment (hpsaturn) [inline src/main.cpp]: The am2320 sensor read lines and print shouldn't be included in `avarageLoop()` method, these lines should be a independent method, like `getHumidityRead()` for example, and this method should called from main loop like `wifiLoop()` for example, with the same validation `v25.size()==0` for send it on each `stime` reached.

Comment (danielbernalb) [reply src/main.cpp]: Done, two new routines:
getHumidityRead()
humidityLoop()

File: src/main.cpp [hunk 2]
```diff
@@ -480,6 +510,9 @@ void setup() {
   gui.displayInit(u8g2);
   gui.showWelcome();
   sensorInit();
+  //////////////
```
Comment (hpsaturn) [inline src/main.cpp]: extra comments here, please remove

File: src/main.cpp [hunk 3]
```diff
@@ -214,15 +244,15 @@ bool influxDbIsConfigured(){
...
  * "id","pm1","pm25","pm10,"hum","tmp","lat","lng","alt","spd","stime","tstp"
- * 
+ *
  */
 void influxDbParseFields(char* fields){
   sprintf(
     fields,
-    "pm1=%u,pm25=%u,pm10=%u,hum=%d,tmp=%d,lat=%d,lng=%d,alt=%d,spd=%d,stime=%i,tstp=%u",
-    0,apm25,apm10,0,0,0,0,0,0,stime,0
+    "pm1=%u,pm25=%u,pm10=%u,hum=%u,tmp=%u,lat=%d,lng=%d,alt=%d,spd=%d,stime=%i,tstp=%u", //cambie de %d a %u para hum y tmp
```
Comment (hpsaturn) [inline src/main.cpp]: Only comments for the code and in english please

File: src/main.cpp [hunk 4]
```diff
@@ -195,6 +210,21 @@ String getFormatData(unsigned int pm25, unsigned int pm10){
...
+ void getHumidityRead(){
+   humidity = am2320.readHumidity();
+   temperature = am2320.readTemperature();
+   Serial.println("-->[AM2320] Humidity: "+ String(humidity) + " % Temperature: " + String(temperature) + " °C");
+   humint = round (humidity);
+   tmpint = round (temperature);
+ }
+
+void humidityLoop(){
+//  if(v25.size()==0) {
```
Comment (hpsaturn) [inline src/main.cpp]: Why not when the 0 is reached?

File: src/main.cpp (hunk 2 above)
Comment (danielbernalb) [reply src/main.cpp]: Removed

File: src/main.cpp (hunk 3 above)
Comment (danielbernalb) [reply src/main.cpp]: Removed

File: src/main.cpp (hunk 4 above)
Comment (danielbernalb) [reply src/main.cpp]: Not worked, only v25.size() >= stime copied from averageLoop()

**Pull request merged**